internal/command/views/hook_ui.go

```diff
@@ -79,8 +79,8 @@ const (
 	uiResourceRead
 	uiResourceNoOp
 	uiResourceOpen
-	uiResourceRenew
-	uiResourceClose
+	// NOTE: Renew and Close do not have a specific operation identifier here
```

Comment:
Do we want to move Open to it's own hook as well?

Reply:
CC @Yantrio ?

Reply:
Done in 5b554df8974f6a9c73a484dbf20fd7f8977644de.
I am not sure that we need that, but before, by using PreApply/PostApply we had the output in the json output too.
Therefore, added the support for these actions in there too. I didn't go all in to add progress logs.

Please let me know how you feel about it: should we add the json hooks too? Should we have also progress ones?

Reply:
I think what you added makes sense.